Next error in step 1 upgrading to rc4

Now I get this error:

```
    cursor.execute(statement, parameters)
sqlalchemy.exc.IntegrityError: (psycopg2.IntegrityError) ERREUR:  la colonne ┬½ auth ┬╗ contient des valeurs NULL
 [SQL: 'ALTER TABLE main_internet2.ogc_server ALTER COLUMN auth SET NOT NULL']
CONST_Makefile:1244: recipe for target 'upgrade1' failed
make: *** [upgrade1] Error 1
```

What is `auth` supposed to be ?

--- a/c2cgeoportal/scaffolds/update/CONST_alembic/main/versions/e004f76e951a_add_missing_not_null.py
+++ b/c2cgeoportal/scaffolds/update/CONST_alembic/main/versions/e004f76e951a_add_missing_not_null.py
@@ -48,13 +48,37 @@ def upgrade():
 
     op.alter_column('layer_wmts', 'url', nullable=False, schema=schema)
     op.alter_column('layer_wmts', 'layer', nullable=False, schema=schema)
+    op.execute(
+        "UPDATE ONLY {schema}.layer_wmts SET image_type = 'image/png' where image_type IS NULL"
+        .format(schema=schema)
+    )
     op.alter_column('layer_wmts', 'image_type', nullable=False, schema=schema)
     op.alter_column('layer_wms', 'ogc_server_id', nullable=False, schema=schema)
     op.alter_column('layer_wms', 'layer', nullable=False, schema=schema)
+    op.execute(
+        "UPDATE ONLY {schema}.layer_wms SET time_mode = 'disabled' where time_mode IS NULL"
+        .format(schema=schema)
+    )
     op.alter_column('layer_wms', 'time_mode', nullable=False, schema=schema)
+    op.execute(
+        "UPDATE ONLY {schema}.layer_wms SET time_widget = 'slider' where time_widget IS NULL"
+        .format(schema=schema)
+    )
     op.alter_column('layer_wms', 'time_widget', nullable=False, schema=schema)
+    op.execute(
+        "UPDATE ONLY {schema}.ogc_server SET image_type = 'image/png' where image_type IS NULL"
+        .format(schema=schema)
+    )
     op.alter_column('ogc_server', 'image_type', nullable=False, schema=schema)
+    op.execute(
+        "UPDATE ONLY {schema}.ogc_server SET type = 'mapserver' where type IS NULL"
+        .format(schema=schema)
+    )
     op.alter_column('ogc_server', 'type', nullable=False, schema=schema)
+    op.execute(
+        "UPDATE ONLY {schema}.ogc_server SET auth = 'Standard auth' where auth IS NULL"
+        .format(schema=schema)
+    )
     op.alter_column('ogc_server', 'auth', nullable=False, schema=schema)
 
 

